onnx/defs/schema.cc

```diff
@@ -844,7 +861,7 @@ OpName_Domain_Version_Schema_Map& OpSchemaRegistry::map() {
       // In debug builds, the number of schema registered in this constructor
       // is compared against the number of calls to schema registration macros.
 #ifndef NDEBUG
-      size_t dbg_initial_schema_count = GetRegisteredSchemaCount();
+      size_t dbg_initial_schema_count = OpSchemaRegistry::GetRegisteredSchemaCount();
```

Comment:
this needs to move back under private right?

Reply:
Updated. Thanks